Document tags automatically added to Synthetic test monitors (#38412)
DataDog/documentation: content/en/monitors/types/synthetic_monitoring.md

- If you have multiple layers of notifications (for example, notifying more teams the longer a Synthetic test is alerting), Datadog recommends enabling [renotification][5] on your Synthetic monitors.

## Automatically added tags

> Datadog adds the following tags to a Synthetic test monitor based on the test's configuration, in addition to any custom tags you add. Use these tags to search and filter on the [{{< ui >}}Manage Monitors{{< /ui >}}][2] page or in the Synthetic Monitoring test list.

Reviewer: ```suggestion
In addition to any custom tags you add, Datadog adds the following tags to a Synthetic test monitor based on the test's configuration. Use these tags to search and filter on the [{{< ui >}}Manage Monitors{{< /ui >}}][2] page or in the Synthetic Monitoring test list.
```

> | Tag key             | Available values                                                                 | What it captures                                                                                                    |

Reviewer: ```suggestion
| Tag key             | Available values                                                                 | What the tag captures                                                                                                    |
```

|----------------------|-----------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|

> | `check_type`         | `api`, `browser`, `api-ssl`, `api-dns`, `api-tcp`, `api-icmp`, `api-grpc`, `api-udp`, `api-websocket`, `api-multi`, `mobile` | The test type, plus the subtype when there is one. The `http` subtype is left off `api` tests for brevity.        |

Reviewer: ```suggestion
| `check_type`         | `api`, `browser`, `api-ssl`, `api-dns`, `api-tcp`, `api-icmp`, `api-grpc`, `api-udp`, `api-websocket`, `api-multi`, `mobile` | The test type and, if applicable, its subtype. The `http` subtype is left off `api` tests for brevity.        |
```

| `check_status`       | `live`, `paused`                                                                   | Whether the test is active or paused.                                                                              |
| `probe_dc`           | `aws:us-east-1`, `aws:eu-west-1`, and other managed or private locations          | The locations the test runs from. Multi-location tests have one `probe_dc` tag for each assigned location.         |

> | `ci_execution_rule`  | `blocking`, `non_blocking`                                                         | The test's CI/CD execution rule, present when the test is used as a quality gate in a CI/CD pipeline.              |

Reviewer: When you say "present when the test is used as a quality gate..." what is "present" referring to? If it's the tag, I suggest:

```suggestion
| `ci_execution_rule`  | `blocking`, `non_blocking`                                                         | The test's CI/CD execution rule. The tag is added when the test is used as a quality gate in a CI/CD pipeline.              |
```

> These tags update automatically when you edit the test, so searching on them stays accurate as tests move locations, get paused, or change their CI/CD configuration. Search on them with the `tag` facet, quoting the full `key:value` pair. For example:

Reviewer: ```suggestion
These tags update automatically when you edit the test, so searches remain accurate as tests move locations, get paused, or change their CI/CD configuration. Search for tags using the `tag` facet and quoting the full `key:value` pair. For example:
```

- `type:synthetics tag:"check_status:live"` finds all active Synthetic test monitors.
- `type:synthetics tag:("probe_dc:aws:us-east-1" AND "probe_dc:aws:ap-northeast-1")` finds tests running from both locations.

> - `type:synthetics tag:"ci_execution_rule:blocking"` finds the tests configured to block a CI/CD pipeline on failure.

Reviewer: ```suggestion
- `type:synthetics tag:"ci_execution_rule:blocking"` finds tests configured to block a CI/CD pipeline if they fail.
```
